### ISSUE TITLE
Validate fails on existing module

### ISSUE THREAD

**Issue by garethr** [puppetlabs/pdk#158]
With PDK installed via gems into the system ruby (2.4.0 in this case). Running validate in a module (puppetlabs-apache here) throws the following error.

```
pdk (INFO): Running all available validators...
[✖] Resolving Gemfile dependencies

/usr/local/lib/ruby/site_ruby/2.4.0/rubygems.rb:270:in `find_spec_for_exe': can't find gem bundler (>= 0.a) (Gem::GemNotFoundException)
        from /usr/local/lib/ruby/site_ruby/2.4.0/rubygems.rb:298:in `activate_bin_path'
        from /usr/local/bundle/bin/bundle:22:in `<main>'
pdk (FATAL): Unable to resolve Gemfile dependencies.
```

Note `pdk` and `bundler` are both installed:

```
gem list

*** LOCAL GEMS ***

bigdecimal (default: 1.3.0)
bundler (1.15.1, 1.14.6)
childprocess (0.6.3)
colored (1.2)
cri (2.9.1)
deep_merge (1.1.1)
did_you_mean (1.1.0)
equatable (0.5.0)
fast_gettext (1.1.0)
ffi (1.9.18)
gettext (3.2.3)
gettext-setup (0.26)
io-console (default: 0.4.6)
json (default: 2.0.2)
json_pure (2.1.0)
locale (2.1.2)
minitest (5.10.1)
necromancer (0.4.0)
net-telnet (0.1.1)
openssl (default: 2.0.2)
pastel (0.7.1)
pdk (0.4.1)
power_assert (0.4.1)
psych (default: 2.2.2)
rake (12.0.0)
rdoc (default: 5.0.0)
rubygems-update (2.6.11)
test-unit (3.2.3)
text (1.3.1)
tty-color (0.4.2)
tty-cursor (0.4.0)
tty-prompt (0.12.0)
tty-spinner (0.4.1)
wisper (1.6.1)
xmlrpc (0.2.1)
```

and `bundle` is on the path:

```
which bundle
/usr/local/bundle/bin/bundle
```

**Comment by DavidS** [comment]
I could reproduce this locally, and trace the reason for the failure:

When installing the pdk as gem, and running the first commands in a new module, https://github.com/puppetlabs/pdk/blob/7448a70db1c224185afe96839698dac5db022ee2/lib/pdk/util/bundler.rb#L140 is executing the commend to install the required gems on a cold cache, and subsequently fails.

**Comment by scotje** [comment]
Should be resolved by #170, will be making a new gem release shortly.
